test/start-stop.make

```diff
@@ -1,68 +1,23 @@
-			: "To make emulated NVDIMMs usable by `ndctl`, we use labels initialization, which must be performed once after the first bootup" ; \
-			: "with new device(s)." ; \
-			_work/$(CLUSTER)/ssh.$$i "/usr/bin/ndctl disable-region region0"; \
-			_work/$(CLUSTER)/ssh.$$i "/usr/bin/ndctl init-labels nmem0"; \
-			_work/$(CLUSTER)/ssh.$$i "/usr/bin/ndctl enable-region region0"; \
-			touch _work/$(CLUSTER)/qemu.$$i.labelsdone; \
-		fi; \
-		_work/$(CLUSTER)/ssh kubectl label --overwrite node host-$$i storage=pmem; \
-	done
-	if ! [ -e _work/$(CLUSTER)/secretsdone ] || [ $$(_work/$(CLUSTER)/ssh kubectl get secrets | grep -e pmem-csi-node-secrets -e pmem-csi-registry-secrets | wc -l) -ne 2 ]; then \
-		KUBECTL="$(PWD)/_work/$(CLUSTER)/ssh kubectl" PATH='$(PWD)/_work/bin/:$(PATH)' ./test/setup-ca-kubernetes.sh && \
-		touch _work/$(CLUSTER)/secretsdone; \
-	fi
-	_work/$(CLUSTER)/ssh kubectl version --short | grep 'Server Version' | sed -e 's/.*: v\([0-9]*\)\.\([0-9]*\)\..*/\1.\2/' >_work/$(CLUSTER)/kubernetes.version
-	( . test/test-config.sh && _work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-csi-$${TEST_DEVICEMODE}-testing.yaml )
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-ext4.yaml
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-xfs.yaml
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-cache.yaml
-	@ echo
-	@ echo "The test cluster is ready. Log in with _work/$(CLUSTER)/ssh, run kubectl once logged in."
-	@ echo "Alternatively, KUBECONFIG=$$(pwd)/_work/$(CLUSTER)/kube.config can also be used directly."
+	test/start-kubernetes.sh && \
+	if ! [ -e _work/$(CLUSTER)/clear-kvm.secretsdone ] || [ $$(_work/$(CLUSTER)/ssh-clear-kvm kubectl get secrets | grep -e pmem-csi-node-secrets -e pmem-csi-registry-secrets | wc -l) -ne 2 ]; then \
+		KUBECTL="$(PWD)/_work/$(CLUSTER)/ssh-clear-kvm kubectl" PATH='$(PWD)/_work/$(CLUSTER)/bin/:$(PATH)' ./test/setup-ca-kubernetes.sh && \
+		touch _work/$(CLUSTER)/clear-kvm.secretsdone; \
+	fi \
+	&& test/setup-deployment.sh
+
+	@ echo "The test cluster is ready. Log in with _work/ssh-clear-kvm, run kubectl once logged in."
+	@ echo "Alternatively, KUBECONFIG=$$(pwd)/_work/kube.config can also be used directly."
 	@ echo "To try out the pmem-csi driver persistent volumes:"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-pvc.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-app.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
+	@ echo "   cat deploy/kubernetes-$$(cat _work/clear-kvm-kubernetes.version)/pmem-pvc.yaml | _work/ssh-clear-kvm kubectl create -f -"
```

Comment:
When you rebased, you removed the usage of `$(CLUSTER)` and brought back the hard-coded `clear-kvm`. Please keep using that variable so that we can prepare and use multiple different clusters in parallel.

Reply:
I added the use of $CLUSTER